jrpc/internal/service.go

```diff
@@ -188,10 +189,28 @@ func (s *jasperService) ResetTags(ctx context.Context, id *JasperProcessID) (*Op
 	if err != nil {
 		err = errors.Wrapf(err, "problem finding process '%s'", id.Value)
 		return &OperationOutcome{
-			Succuess: false,
-			Text:     err.Error(),
+			Success: false,
+			Text:    err.Error(),
 		}, err
 	}
 	proc.ResetTags()
-	return &OperationOutcome{Succuess: true, Text: "set tags"}, nil
+	return &OperationOutcome{Success: true, Text: "set tags"}, nil
+}
+
+func (s *jasperService) DownloadFile(ctx context.Context, info *DownloadInfo) (*OperationOutcome, error) {
+	resp, err := http.Get(info.Url)
```

Comment:
should probably let users pass a client to the service in its constructor rather than using the default http package client one.

Reply:
Added a client to the jasperService struct.